### ISSUE TITLE
Allow WalletConnect integration

### ISSUE THREAD

**Issue by skylarweaver** [rstormsf/multisender#38]
Is there a reason one cannot connect their wallet using WalletConnect? Would be great to have that integrated

**Comment by Zayon986** [comment]
Yes you are right... Please you guys should please try and put that into Consideration.
Thank you ☺
